virtio-devices/src/vhost_user/mod.rs

```diff
@@ -140,9 +140,10 @@ pub struct ReconnectEpollHandler {
     pub acked_protocol_features: u64,
     pub socket_path: String,
     pub server: bool,
+    pub slave_req_handler: Option<MasterReqHandler<S>>,
 }
```

Comment:
I think the name of `ReconnectEpollHandler` should be more generic, as it is now also handling the slave requests.